Source/WebCore/testing/Internals.h

```diff
@@ -513,7 +513,7 @@ class Internals final : public RefCounted<Internals>, private ContextDestruction
     uint64_t storageAreaMapCount() const;
 
     uint64_t elementIdentifier(Element&) const;
-    bool isElementAlive(Document&, uint64_t documentIdentifier) const;
+    bool isElementAlive(uint64_t documentIdentifier) const;
```

Comment:
Not new: The argument name here is wrong, a copy and paste mistake, it’s an element identifier, not a document identifier.